version.sbt

```diff
@@ -0,0 +1 @@
+version in ThisBuild := "3.20.0-SNAPSHOT"
```

Comment:
Mostly it was to move the version declaration its own place since that helps a lot with regex-style code analysis (https://github.com/sbt/sbt-release), but I guess you don't use sbt-release anyways, maybe just sbt publish?
I can move it back to the Build.scala file.